apps/omg_watcher/lib/web/controllers/transaction.ex

```diff
@@ -68,9 +64,9 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
   The endpoint responds with transaction bytes that wallet uses to sign with user's keys. Then signed transaction
   is submitted directly to plasma chain.
   """
-  def encode_transaction(conn, body) do
-    with {:ok, {inputs, outputs}} <- parse_request_body(body),
-         {:ok, transaction} <- State.Transaction.create_from_utxos(inputs, outputs) do
+  def transaction_encode(conn, params) do
```

Comment:
I think we're free to obsolete this, I just wonder can we just delete it just now...
https://github.com/omisego/elixir-omg/pull/264#discussion_r232286520

Reply:
I've posted info on #developers channel - if no serious concerns appear there pls remove